packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -206,76 +207,103 @@ class _NetworkProfilerControlsState extends State<_NetworkProfilerControls>
+            Expanded(
+              child: SearchField<NetworkController>(
+                searchController: controller,
+                searchFieldEnabled: hasRequests,
+                searchFieldWidth: screenWidth <= MediaSize.xs
+                    ? defaultSearchFieldWidth
+                    : wideSearchFieldWidth,
+              ),
+            ),
+            const SizedBox(width: denseSpacing),
+            Expanded(
+              child: StandaloneFilterField<NetworkRequest>(
+                controller: controller,
+                filteredItem: 'request',
+              ),
+            ),
+            const SizedBox(width: denseSpacing),
+            if (FeatureFlags.networkSaveLoad.isEnabled)
+              OpenSaveButtonGroup(
+                screenId: ScreenMetaData.network.id,
+                saveFormats: const [SaveFormat.devtools, SaveFormat.har],
+                gaItemForSaveFormatSelection: (SaveFormat format) =>
+                    switch (format) {
+                      SaveFormat.devtools => gac.saveFile,
+                      SaveFormat.har => gac.NetworkEvent.downloadAsHar.name,
+                    },
+                onSave: (SaveFormat format) async {
+                  switch (format) {
+                    case SaveFormat.devtools:
+                      await controller.fetchFullDataBeforeExport();
+                      controller.exportData();
+                    case SaveFormat.har:
+                      await controller.exportAsHarFile();
+                  }
                 },
-            onSave: (SaveFormat format) async {
-              switch (format) {
-                case SaveFormat.devtools:
-                  await controller.fetchFullDataBeforeExport();
-                  controller.exportData();
-                case SaveFormat.har:
-                  await controller.exportAsHarFile();
-              }
-            },
-          )
-        else
-          DownloadButton(
-            tooltip: 'Download as .har file',
-            minScreenWidthForText: _NetworkProfilerControls._includeTextWidth,
-            onPressed: controller.exportAsHarFile,
-            gaScreen: gac.network,
-            gaSelection: gac.NetworkEvent.downloadAsHar.name,
-          ),
+              )
+            else
+              DownloadButton(
+                tooltip: 'Download as .har file',
+                minScreenWidthForText:
+                    _NetworkProfilerControls._includeTextWidth,
+                onPressed: controller.exportAsHarFile,
+                gaScreen: gac.network,
+                gaSelection: gac.NetworkEvent.downloadAsHar.name,
+              ),
+          ],
+        ),
+        if (!_recording) ...[
+          const SizedBox(height: intermediateSpacing),
+          _NetworkLoggingPausedMessage(screenId: NetworkScreen.id),
+        ],
       ],
     );
   }
 }
 
+class _NetworkLoggingPausedMessage extends BannerInfo {
```

Comment:
I don't know if we are getting much benefit from extending BannerInfo, since it's really designed to be used with the banner controller (that's why we have to pass in parameters that then aren't being used, like the key and dismissOnConnectionChanges)

Instead,  I think we could simply display a `Card` widget here with the "Traffic recording..." text and no X button.